examples/sglang_bench/bench_sharegpt.py

```
@@ -38,6 +38,234 @@
+        # CPATH: pip-installed nvidia packages put nvrtc.h etc. under
+        #   site-packages/nvidia/cuXX/include; add them so TRT-LLM JIT can find them.
+        #   We use the module-level _NVIDIA_CPATH constant (computed from sys.prefix on
+        #   the head node; the path is identical on the peer node via NFS).
+        # FLASHINFER_WORKSPACE_BASE: point peer node JIT cache to local /tmp so the
+        #   peer doesn't share the NFS lock files with the head node, which causes
+        #   OSError: [Errno 116] Stale file handle during FileLock acquisition.
+        conda_bin_dir = os.path.dirname(python_bin)  # .../envs/XXX/bin
+        conda_prefix = os.path.dirname(conda_bin_dir)  # .../envs/XXX
+        base_path = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+        env_vars = (
+            f"CUDA_HOME={shlex.quote(conda_prefix)}"
```

Ruff (0.15.9)

[error] Undefined name `shlex`

(F821)